src/main/java/world/bentobox/bentobox/util/Util.java

```diff
@@ -656,4 +660,18 @@ public static void resetHealth(Player player) {
         double maxHealth = player.getAttribute(Attribute.GENERIC_MAX_HEALTH).getBaseValue();
         player.setHealth(maxHealth);
     }
+
+    /**
+     * Prettify and try to localize an entity type
+     * @param entityType - entity type
+     * @param string - locale to localize to, if possible
+     * @return string
+     */
```

Comment:
```suggestion
     * @since 1.15.2
     */
```
Or `1.16.0`? Do you plan to do a major version or keep it as a minor one?